home-reader/pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>home</artifactId>
+        <groupId>com.softserveinc.ita.homeproject</groupId>
+        <version>${revision}</version>
+    </parent>
+
+    <packaging>jar</packaging>
+    <artifactId>home-reader</artifactId>
+
+    <properties>
+        <json.name>openapi.json</json.name>
+        <image.application.path>homebayraktar/home-reader</image.application.path>
+        <home-application.package>com.softserveinc.ita.homeproject.application</home-application.package>
+    </properties>
+
+    <dependencies>
+
+        <!--Project dependencies-->
+        <dependency>
+            <groupId>com.softserveinc.ita.homeproject</groupId>
+            <artifactId>home-open-api</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>javax.mail</groupId>
+                    <artifactId>mailapi</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <!--Spring dependencies-->
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-beans</artifactId>
```

Comment:
remove dependency (already included in other dependencies, such as starters): 
- spring-beans 
- spring-boot
- spring-boot-autoconfigure
- spring-test
- spring-core